CoreService/Items.psm1

```diff
@@ -156,10 +156,62 @@ Function Get-Item
 	}
 }
 
+function Test-Object
+{
+    <#
+    .Synopsis
+    Checks if the item with the given ID exists.
+	
+	.Notes
```

Comment:
Got it. Want me to do it? Or you'll do it?
On Mar 21, 2016 11:19 AM, "Peter Kjaer" <notifications@github.com> wrote:

> In CoreService/Items.psm1
> <https://github.com/pkjaer/tridion-powershell-modules/pull/15#discussion_r56795404>
> :
>
> > @@ -156,10 +156,62 @@ Function Get-Item
> >  	}
> >  }
> >
> > +function Test-Object
> > +{
> > +    <#
> > +    .Synopsis
> > +    Checks if the item with the given ID exists.
> > +	
> > +	.Notes
>
> Those notes can go, as they are wrong for this method.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/pkjaer/tridion-powershell-modules/pull/15/files/6df18363718fe61a1c81cd454baa3ecb66312d51#r56795404>
>